From 8bfc3566735a193b7d4c0a59e319e0bfd4531144 Mon Sep 17 00:00:00 2001
From: Jianming Qiao <jianming.qiao@bp.renesas.com>
Date: Thu, 30 Aug 2018 13:05:33 +0100
Subject: [PATCH] Remove googletest to reslove configure.ac error

The Makefile requires Google Test to be available when configuring.
However, the Google Test is not shipped together with this package.
Google Test is supposed to be get via Bazel and used by Google
internally. Since we don't need such test feature, it is simply
dropped.

Signed-off-by: Jianming Qiao <jianming.qiao@bp.renesas.com>
---
 configure.ac | 1 -
 1 file changed, 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index aec10cf..7fbe57d 100644
--- a/configure.ac
+++ b/configure.ac
@@ -214,7 +214,6 @@ AX_CXX_COMPILE_STDCXX([11], [noext], [mandatory])
 #   too.
 export CFLAGS
 export CXXFLAGS
-AC_CONFIG_SUBDIRS([third_party/googletest])
 
 AC_CONFIG_FILES([Makefile src/Makefile benchmarks/Makefile conformance/Makefile protobuf.pc protobuf-lite.pc])
 AC_OUTPUT
-- 
2.7.4

